.github/workflows/python-package.yml

```diff
@@ -60,11 +61,7 @@ jobs:
         else
           cmake -DCOMPUTE_BACKEND=cpu .
         fi
-        if [ ${build_os:0:7} == windows ]; then
-          pwsh -Command "msbuild bitsandbytes.vcxproj /property:Configuration=Release"
-        else
-          make
-        fi
+        cmake --build . --config Release
```

Comment:
Should this be only in the windows case?

Reply:
Both work fine.

Reply:
Yes, --config seems to be a no-op on Linux